### PR TITLE
configured logger and added logger methods

### PR DESCRIPTION
**Pull request by liliia-ponomarenko**

### CODE REVIEW

File: src/main/resources/log4j2.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN">
```
Comment (anyutamysko) [inline src/main/resources/log4j2.xml:2]: i think status can be : info

**Review by anyutamysko — APPROVED** [review]
LGTM!

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+logs/*
```
Comment (KostyaNikiforov) [inline .gitignore:4]: Let's add an empty line to each file you create

File: src/main/java/mate/academy/Main.java [hunk 3]
```diff
@@ -6,15 +6,19 @@
...
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error("Can't login user", e);
```
Comment (KostyaNikiforov) [inline src/main/java/mate/academy/Main.java:21]:
```suggestion
            logger.error("Can't login user bob", e);
```

File: src/main/java/mate/academy/service/OrderServiceImpl.java [hunk 4]
```diff
@@ -4,27 +4,27 @@
...
 
     private List<Product> getAllProductsFromShoppingCart(Long userId) {
-        // NOTE: In production ready code this method should fetch data from DB
-        // For test purpose we simplify this method and return dummy data
         Product iphone = new Product("iPhone X", BigDecimal.valueOf(1199));
         Product macBook = new Product("MacBook Air 2020", BigDecimal.valueOf(1399));
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
-        // TODO: add log message about successful fetched data from DB
+        logger.info("data fetched from DB successful");
```
Comment (KostyaNikiforov) [inline src/main/java/mate/academy/service/OrderServiceImpl.java:27]:
```suggestion
        logger.info("data fetched from DB successful for user with id {}", userId);
```

**Review by KostyaNikiforov — CHANGES_REQUESTED** [review]

**Review by KostyaNikiforov — APPROVED** [review]

**Pull request closed without merging**